data-models/common/acceleration/schema.json

```
@@ -0,0 +1,30 @@
+{
+    "$schema": "http://json-schema.org/draft-07/schema#",
+    "$id": "https://raw.githubusercontent.com/akenza-io/device-type-library/main/data-models/location/acceleration/schema.json",
+    "title": "Acceleration",
+    "description": "Acceleration measurement.",
+    "type": "object",
+    "$defs": {
+        "accX": {
```

```suggestion
        "accelerationX": {
```

Same as already mentioned for `celsius`.